lib/posts/UserPosts.ts

```diff
@@ -94,7 +94,7 @@ export class GraphQLUserPosts implements UserPosts {
               createdAt: creationDate.toISOString(),
               userId
             })
-            .then((value: any) => value.getPost)
+            .value.then((value: any) => value.getPost)
```

Comment:
Let's make sure nothing else breaks now that we need to call .value of the graphql client.

Reply:
Can confirm this to be the case, better that this change got introduced now rather than down the line where we would've had a lot more code using the client.